docs/docs/guides/contributing-components.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Contributing Components
+sidebar_label: Contributing Components
+---
+
+:::warning
+
+This functionality is under development. With these docs, we aim to elicit feedback from our community (you!) and focus our development efforts. Currently, we do not offer support for 3rd party components, but we are working toward supporting this.
+
+:::
+
+Airy's functions are provided by components ready for _plug and play_. With every [installation of Airy](../getting-started/installation/introduction.md) we bundle a few default components to get started:
+
+1. airy-controller
+2. api-admin
+3. api-communication
+4. api-websocket
+5. frontend-inbox
+6. frontend-control-center
+
+Airy's real value comes from the additional components you install through our catalog in the Control Center UI of your Airy Instance.
+In the following, we will explain how to create, update and store components.
+
+## Components
+
+import ComponentDefinition from '../getting-started/componentDefinition.mdx'
+
+<ComponentDefinition/>
+
+The Helm package and information on each component (except core components) are stored in an external repository managed by Airy called [airy-components](https://github.com/airyhq/airy-/airy-components). This repository is made up of directories where each directory contains a component's Helm package and information.
+
+## The Component File Structure
+
+Below is a model of the file structure of a single component inside the [`airy-components`](https://github.com/airyhq/airy-components) repository.
+
+```
+airy-components/
+└── [COMPONENT_NAME]/
+    ├── information.yaml
```

Comment:
I think it was `description.yaml` in the previous text?

Reply:
Yes! Good catch. Will fix